src/data/data1d/filter1d/lorentzToGauss.ts

```diff
@@ -0,0 +1,61 @@
+import { Datum1D } from '../../types/data1d';
+
+export const id = 'lorentz-to-gauss';
+export const name = 'Lorentz-to-Gauss Window';
+
+// This window function is based on https://spin.niddk.nih.gov/NMRPipe/ref/nmrpipe/gm.html
+export function apply(datum1D: Datum1D, options: any = {}) {
+  if (!isApplicable(datum1D)) {
+    throw new Error('LG not applicable on this data');
+  }
+  const { center = 0, GB: gaussHz = 0, LB: expHz = 0 } = options;
```

Comment:
Why do we need to rename the variables (moreover uppercase for a variable). I think they should be named everywhere gaussHz and expHz.

Reply:
@lpatiny @jobo322 

can you let me know what are the new options that we need to add to the header?

Reply:
We need a drop menu with the name of the functions, At the moment we have only `exponential` and `lorentz-to-gauss` 

- exponential only need one value:

`expHz` number [-inf, inf]

- lorentz-to-gauss needs three parameters

`center` number in 0-1
`gaussHz` number in 0-1
`expHz` number [-inf, inf]